tests/integration/catalogue/test_product.py

```diff
@@ -104,6 +114,43 @@ def test_child_products_are_not_part_of_browsable_set(self):
             structure=Product.CHILD)
         self.assertEqual(set([self.parent]), set(Product.objects.browsable()))
 
+    def test_child_products_attribute_values(self):
+        product = Product.objects.create(
+            product_class=self.product_class, parent=self.parent,
+            structure=Product.CHILD)
+
+        self.parent.attr.first_attribute = "klats"
+        product.attr.second_attribute = "henk"
+        self.parent.save()
+        product.save()
+
+        product = Product.objects.get(pk=product.pk)
+        parent = Product.objects.get(pk=self.parent.pk)
+
+        self.assertEqual(parent.get_attribute_values().count(), 1)
+        self.assertEqual(product.get_attribute_values().count(), 2)
+        self.assertEqual(True, hasattr(parent.attr, "first_attribute"))
```

Comment:
`assertTrue`/`assertFalse` is preferred for boolean tests instead of `assertEqual`.